src/routes/liquidity/liquidity-container.tsx

```diff
@@ -13,23 +13,26 @@ export const LiquidityContainer = ({ state }: { state: LiquidityState }) => {
   const { ethAddress } = useWeb3();
   return (
     <section className="liquidity-container">
-      <Callout icon={<Error />} intent="error" title={t("lpEndedTitle")}>
+      <h2>{t("liquidityRewardsTitle")}</h2>
+      <p>
+        {t("liquidityOnsenIntro")}
+        <a href={Links.SUSHI_ONSEN_WHAT_IS}>{t("liquidityOnsenHowItWorks")}</a>
```

Comment:
Could these links be in list form, or some other way to format "how it works" and "FAQ"? That is - if they're formatted like the screenshot, and they could use some breathing room in there

Reply:
Done and updated screenshot